src/app/share-preview/components/share-preview/share-preview.component.html

```diff
@@ -30,9 +30,9 @@
   <pr-breadcrumbs></pr-breadcrumbs>
 </div>
 <div class="share-preview-content" [ngClass]="{'navigating': isNavigating}">
-  <router-outlet></router-outlet>
+  <router-outlet (activate)="subscribeToItemClicks($event)" (deactivate)="unsubscribeFromItemClicks()"></router-outlet>
```

Comment:
TIL about `(activate)` and `(deactivate)` events on `router-outlet`! That's really useful!